### ISSUE TITLE
Add new types with default categories (forager, freegan)

### ISSUE THREAD

**Issue by ezwelty** [falling-fruit/falling-fruit-web#520]
New (and thus pending) types added via the location form's Types input have no categories assigned, so they do not appear on the map by default.

Once we sort out category filtering, the submitted type's categories could be those of the currently selected categories, but I think a good default behavior for now is to simply add default categories (`forager`, `freegan`) to new types.

**Comment by kmahesh18** [comment]
hii @ezwelty , i would like to work on this .. can you assign me?

**Comment by ezwelty** [comment]
@kmahesh18 Welcome and thanks for offering to help. I've assigned you the issue.

**Comment by wbazant** [comment]
I wonder if we might need to temporarily assign all categories to a new type, just to be sure it shows up. I'm working on the categories now so I'll figure it out as part of it.

**Comment by wbazant** [comment]
@ezwelty shall we resolve this by changing the default selection to "forager or freegan category or pending review"? I got hit by the 'pending review isn't on by default' when checking on my orchard planting (https://beta.fallingfruit.org/map/@55.8276605,-4.2512007,18z) and thinking I missed a tree.

**Comment by ezwelty** [comment]
@wbazant Nah, because in pending review are typically lots of new types added via a big municipal tree inventory import that need to be reviewed but are likely of no interest to foragers. These are created as uncategorized. New pending types added via foraging map or dumpster diving map import are assigned `forager` or `freegan` depending on context, and those added by users via the legacy website are assigned `forager, freegan`. Which is why I suggested to add default categories (forager, freegan) to new types created by users via the beta website.

**Comment by wbazant** [comment]
Ah right I can just add the categories to the Post request
